USca/DbManager/Tags/TagsDashboard.xaml.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Windows;
+using System.Windows.Input;
+using USca_DbManager.Util;
+
+namespace USca_DbManager.Tags
+{
+	public partial class TagsDashboard : MyPage
+    {
+        public ObservableCollection<TagDTO> Tags { get; set; } = new();
+        public TagDTO? SelectedTag { get; set; } = null;
+
+        public TagsDashboard()
+		{
+			InitializeComponent();
+            LoadAllTags();
+            DataContext = this;
```

Comment:
Both get applied but since they're the same thing, nothing happens. It's better to set it in the .xaml so that we can get better intellisense. [ec726f1](https://github.com/magley/untitled-scada-project/pull/18/commits/ec726f1a821ad0f44f390366c99b891b5a63b7fe)